### PR TITLE
PP-8532: Use fargate update transactions task

### PR DESCRIPTION
**Pull request by jfharden**
Update transaction updater to use the new fargate version of the task.

### CODE REVIEW

File: src/web/modules/transactions/update/update.http.ts [hunk 1]
```diff
@@ -50,7 +50,7 @@ const uploadToS3 = async function uploadToS3(content: string, user: any): Promis
     const key = (user && user.username || '') + moment().format('x')
     logger.info(`Uploading transactions file to S3 with key ${key}`)
     const response = await s3.putObject({
-      Bucket: aws.AWS_S3_UPDATE_TRANSACTIONS_BUCKET_NAME,
+      Bucket: aws.AWS_S3_UPDATE_TRANSACTIONS_V2_BUCKET_NAME,
```
Comment (katstevens) [inline src/web/modules/transactions/update/update.http.ts:53]: Maybe we could call this env var `AWS_S3_UPDATE_TRANSACTIONS_FARGATE_BUCKET_NAME`? As you've probably noticed by now I am reluctant to put `v2` into actual code...

Comment (jfharden) [reply src/web/modules/transactions/update/update.http.ts:53]: I guess so. I kinda wish we had them named the same in the repos. It means the names of deployed things really don't match the names of the things that define their deployments.....but we're already there and at least it's consistent at the moment.

Comment (jfharden) [reply src/web/modules/transactions/update/update.http.ts:53]: Updated https://github.com/alphagov/pay-infra/pull/3249 with the new variable names, will update this now too

Comment (jfharden) [reply src/web/modules/transactions/update/update.http.ts]: Sorted

**Comment by jfharden** [comment]
Tested this in test-12 and it works :)

**Review by katstevens — APPROVED** [review]
Good spot on the typo 👍

**Pull request merged**